lists/cm/cm-niu.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": {
+    "en": "Tax Identification Number (Cameroon)",
+    "local": "Numéro d'Identifiant fiscal Unique (NIU)"
+  },
+  "url": "https://www.impots.cm/",
+  "description": {
+    "en": "The Directorate General of Tax (Direction Générale des Impôts) of Cameroon issues a unique tax identification number (Numéro d'Identifiant fiscal Unique, or NIU) to all individuals and legal persons liable to pay tax in the country. "
+  },
+  "coverage": [
+    "CM"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "charity",
+    "company",
+    "trust",
+    "unincorporated_body"
+  ],
+  "sector": [],
+  "code": "CM-NIU",
+  "confirmed": true,
+  "deprecated": false,
+  "listType": "primary",
```

Comment:
I'd make this a secondary listType, based on the schema entry stating that 

> [Examples of secondary lists are VAT or Tax identifiers](https://github.com/org-id/register/blob/master/schema/codelist-listType.json#L19)

Reply:
Oops - yup.